server/protocols/jmap-integration-testing/jmap-integration-testing-common/src/test/java/org/apache/james/jmap/methods/integration/cucumber/DownloadStepdefs.java

```diff
@@ -110,7 +110,7 @@ public void appendBigMessageToMailbox(String user, String mailbox, String messag
 
         ComposedMessageId composedMessageId = mainStepdefs.mailboxProbe.appendMessage(user, mailboxPath,
             AppendCommand.from(new ByteArrayInputStream(
-                Strings.repeat("header: 0123456789\r\n", 1024 * 1024)
+                Strings.repeat("header: 0123456789\r\n", 128 * 1024)
```

Comment:
I guess `DownloadGet.feature` uses this test. The original scenario:
```
  Scenario: Getting a message blob previously stored
    Given "alice@domain.tld" mailbox "INBOX" contains a big message "1"
    When "alice@domain.tld" downloads "1"
    Then the blob size is 20971520
```
Should we update blob size to fit with the updated code.

Reply:
The purpose of that tests is testing with a big message, does reducing message size satisfy the original purpose?

Reply:
No, as the name does not suggest it, it do ensure that COntent-Length is specified before download.